cscs-checks/microbenchmarks/fft/fftw_benchmark.py

```diff
@@ -0,0 +1,58 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.required_version('>=2.16-dev0')
+@rfm.parameterized_test([False], [True])
```

Comment:
Please change either how you enable mpi (with a string argument) or create the test name manually, since the automatic naming creates `FFTWTest_True, FFTWTest_False`.